api/blueprints/routes.py

```diff
@@ -13,6 +13,24 @@
 bp = Blueprint('routes', __name__)
 
 
+def format_path(base, **kwargs):
+    path = f'{API_BASE_PATH}/{base}?'
+    for param, value in kwargs.items():
+        path += f"&{param}={value}"
```

Comment:
Encoding and decoding query strings (also referred to as url params) comes with a lot of pitfalls when it comes to encoding strings. In other programming languages you tend to see an `url` class in the std lib that helps constructing these. The closest thing I found for Python is [furl](https://github.com/gruns/furl/). There also seems to be `urllib.parse` for decoding. While it will probably do no harm right now, constructing query strings like this _will_ lead to problems when interfacing with apis that accept utf-8 strings.

Reply:
i suspected that i might not be this easy :) i want to do this properly and using `furl` seems to be a good solution (and a bit more straightforward than using `urllib.parse`).